apps/www/app/components/Header.tsx

```diff
@@ -34,7 +47,19 @@ export function Header() {
             <DiscordLogoIcon className="size-5" />
           </Button>
         </a>
-        <div className="h-5 border-[0.2px] mx-2" />
+        {!!projectId && (
```

Comment:
🤔 It would be great to do something like a `data-hide` prop for the dashboard app, the rendering with it looks much more readable, of course, this doesn't apply to this PR,  just would like to know what you think about it